README.md

```diff
@@ -1,8 +1,65 @@
+# Country By Country Reporting Frontend
 
-# country-by-country-reporting-frontend
+## Info
 
-This is a placeholder README.md for a new repository
+This service allows a user to upload the correct tax file which further goes for validation and once validated it finally gets uploaded.
 
-### License
+This service has a corresponding back-end service, namely country-by-country-reporting.
 
-This code is open source software licensed under the [Apache 2.0 License]("http://www.apache.org/licenses/LICENSE-2.0.html").
\ No newline at end of file
+### Dependencies
+
+| Service           | Link                                                          |
```

Comment:
Add other cbc services such as backend stubs and cbc registration frontend, backend and stubs